src/platform/Makefile.am

```diff
@@ -227,6 +227,44 @@ libDeviceLayer_a_SOURCES               += \
 
 endif #CONFIG_BLE_PLATFORM_BLUEZ
 
+if CHIP_ENABLE_GIO
```

Comment:
maybe you should use CONFIG_WIFI_PLATFORM_WPA instead of this generic CHIP_ENABLE_GIO

Reply:
Ack